src/java.base/share/classes/jdk/internal/classfile/impl/DirectCodeBuilder.java

```diff
@@ -1471,4 +1474,64 @@ public CodeBuilder tableswitch(int low, int high, Label defaultTarget, List<Swit
         writeTableSwitch(low, high, defaultTarget, cases);
         return this;
     }
+
+    @Override
+    public CodeBuilder iload(int slot) {
```

Comment:
Can you put them in the same order they are declared in `CodeBuilder`, like `iload` between `iinc` and `imul`?